### PR TITLE
REVIEW ONLY: Integrate digitalrebar-ux into the revproxy container.

### PR DESCRIPTION
**Pull request by VictorLowther**
Arrange for the digitalrebar-ux repo to be included in the revproxy
container by default, and arrange for the --dev flag to docker-admin to
allow a locally checked out copy to overwrite it.

### CODE REVIEW

File: containers/rebar-rev-proxy/entrypoint.d/25-build-ux.sh [hunk 1]
```diff
@@ -4,11 +4,9 @@
 (
     with_local_proxy
     cd /opt/digitalrebar-ux
-    if [[ $REVPROXY_REBUILD_BOWER ]] ; then
+    if [[ ! -d bower_components ]]; then
         bower --allow-root install --config.interactive=false
         npm install cssnano-cli html-minifier uglify-js
```
Comment (galthaus) [inline containers/rebar-rev-proxy/entrypoint.d/25-build-ux.sh]: This should probably remove .using_prebuilt to force a rebuild,  I think.

Comment (galthaus) [inline containers/rebar-rev-proxy/entrypoint.d/25-build-ux.sh]: yeah - nvm.  Read it wrong.

**Comment by galthaus** [comment]
Needs a rebase.  :-(

**Comment by VictorLowther** [comment]
Already merged.

**Pull request closed without merging**